site/en/blog/chrome-109-beta/index.md

```diff
@@ -0,0 +1,106 @@
+
+## CSS
+
+### Auto range support for font descriptors inside `@font-face` rule
+
+Variable fonts provide users the opportunity to choose how heavy or slanted or wide the typeface should be, using the `font-weight`, `font-style`, and `font-stretch`' descriptors inside the `@font-face` rule. The CSS Working Group added a new value of `auto` for these descriptors, which is now the initial value. This value is implemented in 109 to match the new specification.
+
+### CSS `lh` Length Unit
+
+The `lh` CSS [`<length>`](https://developer.mozilla.org/docs/Web/CSS/length) unit is equivalent to the computed value of the line-height property on the element on which it is used. This allows a `<textarea>` to be given a height equivalent to the number of lines of text expected.
+
+### CSS `hyphenate-limit-chars` property
+
+The `hyphenate-limit-chars` property specifies the minimum number of characters in a hyphenated word. When applying the hyphenation, the optimal minimum number of characters in the word, before the hyphen, or after the hyphen can vary by the design of the page, or by the language. This property allows finer grained control of the hyphenation for better typography on the web. It can also help international pages when the default settings are not optimal. 
+
+### Snap border, outline and column-rule widths before layout
+
+Currently Blink snaps the border widths at paint time. This can cause a visible 1px gap between a parent element's border and a child's background when the border is rounded down during paint, since the snapping floors the value but the layout rounded it up to calculate the child’s position (for example, border-width is set to 10.75 pixels, rounded to 10px at paint time, but 11 at layout time). By making this change Blink will behave like Gecko and WebKit, improving interoperability.
+
+## MathML
+
+Chrome 109 supports [MathML](https://developer.mozilla.org/docs/Web/MathML) Core, a language for describing mathematical notation embeddable in HTML and SVG. MathML is rendered in a CSS-compatible way with OpenType MATH and exposed via platform accessibility APIs. MathML styling is enabled by CSS features including those dedicated to math layout:
+
+The [`math-depth`](https://developer.mozilla.org/docs/Web/CSS/math-depth) property.
+The [`math-shift`](https://developer.mozilla.org/docs/Web/CSS/math-shift) property.
+The [`math-style`](https://developer.mozilla.org/docs/Web/CSS/math-style) property. 
+The `math` value for the [`display`](https://developer.mozilla.org/docs/Web/CSS/display) property
+The `math` `font-family` name
+The `math-auto` value for the [`text-transform`](https://developer.mozilla.org/docs/Web/CSS/text-transform) property. 
+
+The `MathMLElement` interface provides a convenient way to manipulate MathML in scripts. 
+
+## Web APIs
+
```

Comment:
Somehow this was dropped. We are publishing [a new blog post for this](https://github.com/GoogleChrome/developer.chrome.com/pull/4478).

```suggestion

### Secure Payment Confirmation on Android Chrome

[Chrome 109 on Android supports Secure Payment Confirmation (SPC)](/blog/spc-on-android), which is a [proposed web standard](https://www.w3.org/TR/secure-payment-confirmation/) that allows customers to authenticate with a credit card issuer, bank, or other payment service provider using a platform authenticator — typically activated with a device's screen unlock feature such as a fingerprint sensor. This usually happens during a payments authentication protocol such as [EMV 3-D Secure](https://www.emvco.com/emv-technologies/3d-secure/) or [Open Banking](https://standards.openbanking.org.uk/). EMV 3-D Secure, for example, has support for SPC in its [v2.3 spec release](https://www.emvco.com/emv_insights_post/what-is-new-with-emv-3ds-v2-3/). We [previously announced](/articles/secure-payment-confirmation/) that SPC was launched for Google Chrome on macOS and Windows and provided developer guides for both [registration](/articles/register-secure-payment-confirmation/) and [authentication](/articles/authenticate-secure-payment-confirmation/).

```

Reply:
@agektmr this isn't on the Chrome Status roadmap therefore isn't in the post. So it was't dropped, it wasn't in the data. If you want things to be included in the beta post, ensure they are showing up on the roadmap.